backend/app/assets/stylesheets/spree/backend/globals/_variables.scss

```diff
@@ -206,3 +207,14 @@ $z-index-navbar-flyout:          1000 !default;
 // Sidebar
 //--------------------------------------------------------------
 $border-sidebar:                 1px solid $color-sidebar-border !default;
+
+// Media queries
+//--------------------------------------------------------------
+$desktop-width: 959px;
+$tablet-width: 768px;
+$mobile-width: 480px;
+
+$large-device: "only screen and (min-width: #{$tablet-width})";
+$desktop: "only screen and (max-width: #{$desktop-width})";
+$tablet: "only screen and (max-width: #{$tablet-width})";
+$mobile: "only screen and (max-width: #{$mobile-width})";
```

Comment:
@graygilmore for now we just converted the skeleton media queries [set here](https://github.com/solidusio/solidus/blob/master/backend/app/assets/stylesheets/spree/backend/shared/_skeleton.scss#L79). We internally discussed about changing them to a mobile first approach but it seemed like something not related to this PR and probably something that need a better testing.